### PR TITLE
Mclyte

### PR DESCRIPTION
**Pull request by McLyte**

### CODE REVIEW

**Comment by McLyte** [comment]
I spent hours trying to change the author name(another account was signed in on my PC, which I did not realize until I made a commit) on my Visual Code Studio. Eventually, GitLens (thanks to Google & Stack overflow) was the solution to my almost 5 hours ordeal

**Comment by MridulS** [comment]
Thanks @McLyte!

Those 5 hours were well spent :)

File: 2022-round-1/McLyte/nx_version.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+2.6.3
```
Comment (MridulS) [inline 2022-round-1/McLyte/nx_version.txt:1]: Could you update this to the latest version of networkx, 2.7.1?

Please also do check your python version.

**Comment by MridulS** [comment]
Thanks for opening this PR! I'll close this for now :)

**Pull request closed without merging**